jlibra-core/src/main/java/dev/jlibra/admissioncontrol/transaction/AddressArgument.java

```diff
@@ -18,4 +23,11 @@ public Type type() {
         return Type.ADDRESS;
     }
 
+    @Override
```

Comment:
I originally thought about using the GrpcMapper to create the grpc objects and keep the value objects simple value holders, but I can also see the benefits of putting the mapping logic in the value objects, this might make it easier to find the mapping logic instead of looking for it in the big mapper class.

But should we use this same pattern for all of them and get rid of the GrpcMapper, this is also something we can create an issue about, I just want to hear your thought on this change.

Reply:
I don't know what's best but i like to have `X toX()` or `fromY(Y x)` directly in the subject class. It simplifies greatly implementation.
After all first mapper `toString()` is in the Object directly. I see many people who like "mapper" classes. Where does this pattern come from, different language influence maybe?